pkgs/os-specific/linux/zfs/unstable.nix

```diff
@@ -23,9 +23,10 @@ callPackage ./generic.nix args {
   # IMPORTANT: Always use a tagged release candidate or commits from the
   # zfs-<version>-staging branch, because this is tested by the OpenZFS
   # maintainers.
-  version = "2.2.0-rc5";
+  version = "2.2.1-unstable-2023-10-21";
+  rev = "95785196f26e92d82cf4445654ba84e4a9671c57";
```

Comment:
The comment says explicitly to use a ZFS staging ref, not a sha1 git object?

Reply:
How do you propose to get a reproducible rev from a branch, if not a git commit SHA? There is no tag for this staging branch...

And the comment has the words *or commits from the zfs-<version>-staging branch*

Reply:
You are totally right... Apologies, I must be very tired.